unix.go

```diff
@@ -0,0 +1,11 @@
+package adb
+
+/*
+Exported Access function calls private access function.
```

Comment:
Function comment should start with the name of the function. The fact that it's exported is implied by the name, no need to say it.